regulations/static/regulations/css/scss/partials/_reg_text.scss

```diff
@@ -17,25 +17,54 @@
     }
 }
 
-.reg-section {
+article {
     max-width: 75ch;
-    margin: $spacer-2;
+    margin: $spacer-4;
 
-    &:target {
-        background: $teritary_background_color;
-        padding: $spacer-1 0;
-        scroll-margin-top: $spacer-4;
+    i {
+        font-weight: bold;
+    }
+
+    h1 {
+        margin-left: -$spacer-6;
+        padding-left: $spacer-3;
+    }
+
+    p {
+        margin: $spacer-3 0;
+        padding: 0 $spacer-3 0 $spacer-3;
+
+        @for $i from 3 through 10 {
+
+            &.depth-#{$i} {
+                padding-left: $spacer-3 + $spacer-3 * ($i - 2);
```

Comment:
I know I ended up changing this from `$spacer-3 + 16px` to conform to the design system stuff, but we could actually just write this as `$spacer-6` now that I look at it.